ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -313,6 +319,24 @@ public static SourceName getCommandsStreamName() {
   @Override
   public void startAsync() {
     log.debug("Starting the ksqlDB API server");
+
+    final KsqlResourceContext ksqlResourceContext
+            = new KsqlResourceContextImpl(ksqlConfigNoPort, restConfig);
+
+    boolean isFipsValidatorConfigured = false;
+    for (KsqlResourceExtension resourceExtension : ksqlResourceExtensions) {
+      resourceExtension.register(ksqlResourceContext);
+      if (KsqlConstants.FIPS_VALIDATOR
+              .equals(resourceExtension.getClass().getCanonicalName())) {
+        isFipsValidatorConfigured = true;
+      }
```

Comment:
we can get rid of FIPS Specific checks. We can move tests related to that to confluent-security-plugins itself.